t/predictable-errors.t

```diff
@@ -0,0 +1,33 @@
+use strict;
+use warnings;
+
+use JSON::Validator;
+use Math::Permute::List;
+use Test::More;
+
+my $jv = JSON::Validator->new();
+permute {
+    $jv->schema(my $schema_text = join('',
+       '{ "type": "object", "properties": {',
+       join(', ', @_),
+       '}}'
+    ));
```

Comment:
You don’t have to give it a string. Just pass in the data structure instead.

Reply:
I don't have control over the order of the properties in a perl data structure, I do it this way in case the order the JSON parser sees them makes a difference to the order in which they are processed.